source/issues/crashes.rst

```diff
@@ -6,7 +6,7 @@ Crashes
 PocketMine-MP may crash for a number of reasons:
 
 - Bad / faulty plugins.
-- Running out of memory due to not enough or memory leak.
+- Running out of memory, or memory leaks.
```

Comment:
I think the original statement here was correct, no?

Reply:
The grammar didn't sit well with me, so I opted to remake it. But your suggestion of `Running out of memory, due to too-low limit or a memory leak` in discord sounds a lot better.